crates/agent/src/controllers/materialization.rs

```diff
@@ -283,6 +287,7 @@ fn is_false(b: &bool) -> bool {
 /// Status information about the `sourceCapture`
 #[derive(Debug, Clone, Serialize, Deserialize, Default, PartialEq, JsonSchema)]
 pub struct SourceCaptureStatus {
+    pub source_capture: SourceCaptureDef,
```

Comment:
It doesn't seem necessary or desirable to duplicate the `SourceCaptureDef` on the status. The live spec has it, and should be the source of truth.